03-seeking-help.Rmd

```diff
@@ -124,6 +124,12 @@ your issue.
 > What is the difference between the `sep` and `collapse` arguments?
 > 
 
+> ## Challenge 3 {.challenge}
+> Use help to find a function (and it's assosiated parameters) that you could use
+> to load data from a csv file in which columns are delimited with "\t" (tab) and
+> decimal point is a "." (period).
```

Comment:
"decimal point" -> "the decimal point"

I would also add a short bit to the effect of:
"This check for decimal separator is important, especially if you are working with international colleagues, because different countries have different conventions for the decimal point (i.e. comma vs period)."